README.md

```diff
@@ -12,6 +12,19 @@ The [`tinywallet`](./tinywallet) package contains a wallet based on the
 Each package may be installed from the [Python Package Index](https://pypi.org/)
 using the [`pip`](https://pip.pypa.io/) command as usual.
 
+## Requirements
+
+To run tinydecred on your machine you will need to set up the following. 
+
+- pip install websocket_client
+- pip install blake256
+- pip install base58
+- pip install PyNaCl
+- pip install appdirs
+
+Set up and run dcrd. Ensure your rpcusername and rpcpassword is set in your dcrd config file.
+Installation guide can be found [`here`](https://docs.decred.org/wallets/cli/dcrd-setup/).
```

Comment:
dcrd is completely optional, and it's integration is very shallow right now. I wouldn't say much about dcrd yet.